mlflow/tensorflow.py

```diff
@@ -719,10 +720,6 @@ class __MLflowTfKeras2Callback(Callback):
         Records model structural information as params when training starts.
         """
 
-        def __init__(self, log_models):
```

Comment:
Note: Since `log_models` is not added to `self`  in `__init__` anymore, removing the entire method so that super's init is invoked automatically without running into `Useless super delegation in method` linting error.